lib/poxa/web_hook/event_table.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Poxa.WebHook.EventTable do
+  @table_name :web_hook_events
+
+  def new do
+    :ets.new(@table_name, [:bag, :public, :named_table])
+  end
+
+  def init do
+    case :ets.info(@table_name) do
```

Comment:
I did this because this is called when the handler starts. In the case the handler crashes, it would call this function again and crash as the table already exists.